guardian/front-press-monitor
# Upgrade node versions

## What does this change?



This PR upgrades node versions to version 16.

Previously, node version 12 was defined in the .nvmrc, and version 14 was defined in cloudformation as the lambda runtime environment. This PR upgrades them to the same version.

Node 14 will be [deprecated](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) for AWS lambdas on Nov 27, 2023, hence the need to upgrade.

Upgrading to node 18 is not as straightforward an upgrade, and requires making changes to the application code, hence upgrading to version 16 for now.

## How to test



Steps to test:
- run `yarn build` - this should run successfully
- run `yarn test` - all tests should pass
- the CI build should run successfully
